### DISCUSSION TITLE
[Feature Request] Enable an RSS icon in the URL bar when an RSS feed is detected, like in older Firefox

### DISCUSSION THREAD

**Discussion by MMachado05** [zen-browser/desktop#3072] [Ideas]
I'm a *huge* fan of RSS, and I'm so for good reason. Reading the news on the internet is a mess, where you have to bookmark every separate news source and remember to check them individually, and then load up an add-ridden website that's probably covered in psychologically-alluring "related stories" to keep you doomscrolling for a while, with the non-RSS alternative being polluting your email inbox with a bunch of newsletters. I use the Newsflash GNOME app to get up-to-date with my news, and it's so much *quieter*, and it's much easier to simply open the app, catch up one what's happened, and close it, without getting distracted by other emails or news stories I'm being convinced are important.

In the past (like, *more than a decade* in the past), Firefox would show a little icon when it detected that the website you had visited had an RSS feed:
![image](https://github.com/user-attachments/assets/44922006-a197-46ac-88cf-fc0f57967b1e)

This was removed, much to the chagrin of some users, on account of it being "not very popular," but I don't find myself agreeing with the notion that a mild lack of popularity could justify removing such a non-intrusive yet useful tool. Heck, if you *don't* know what RSS is, maybe the icon could be hovered-over so that curious users could discover what RSS can provide them.

I think Zen has the unique opportunity to re-implement this as a native functionality. Zen is all about staying out of your way and boosting your productivity, and RSS is *incredibly* good at doing this. Yes, you could get an extension or a mod, but I figure the implementation might be really simple (at least compared to other Zen features) and it would have the added benefit of exposing those who are unfamiliar with RSS to its advantages, and may actually bring the curious user some added productivity benefits.

Let me know what y'all think! This is pretty similar to [this discussion](https://github.com/zen-browser/desktop/discussions/769), but I don't think there should be a reader in Zen, necessarily; just an easy way to grab the feed URL and put it in a feed reader of your choice.

**Comment by 5onderling** [comment]
Oh, I guess what I wrote in the other discussion fits actually better here, but the other has more upvotes, so I just link it.
https://github.com/zen-browser/desktop/discussions/769#discussioncomment-14503048
